pod/playlist/views.py

```diff
@@ -366,6 +366,11 @@ def handle_post_request_for_add_or_edit_function(request, playlist: Playlist) ->
         if playlist
         else PlaylistForm(request.POST)
     )
+    if playlist:
+        page_title = _("Edit the playlist") + f' "{playlist.name}"'
```

Comment:
ici il faudrait plutôt écrire ceci pour une traduction correcte dans toutes les langues (pensons aux langues qui se lisent de droite à gauche) :

`page_title = _("Edit the playlist “%(pname)s”") % {"pname": playlist.name}`